todo_app/data/trello_items.py

```diff
@@ -0,0 +1,76 @@
+import requests
+import os
+
+class Item:
+    def __init__(self, id, name, description, due, status = 'To Do'):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.due = due
+        self.status = status
+
+    @classmethod
+    def from_trello_card(cls, card, list):
+        return cls(card['id'], card['name'], card['desc'], card['badges']['due'], list['name'])
+
+cardsURL = "https://api.trello.com/1/cards/"
+boardsURL = "https://api.trello.com/1/boards/"
+listsURL = "https://api.trello.com/1/list/"
+
+def get_parameters():
+    return {'key': os.getenv('KEY'), 'token': os.getenv('TOKEN')}
+
+def get_lists():
+    return requests.get(boardsURL + os.getenv('BOARD_ID') + "/lists", params=get_parameters())
+
+def get_cards_on_list(listID):
+    return requests.get(listsURL + listID + '/cards', params=get_parameters())
+
+def get_list_id(name):
+    lists = get_lists().json()
+    for list in lists:
```

Comment:
This function can be two lines:
```suggestion
    return next((list['id'] for list in lists if list['name'] == name), None)
```